markdown/3.0/en/ff-suggest.md

```diff
@@ -81,16 +80,64 @@ rather than the rendered: "**Back**packs".
 
 See [Template Engine](documentation/template-engine) for more details.
 
+### Rendered HTML
+
+Container templates provided inside `<ff-suggest>` tag will be rendered inside
+several additional wrapping `<div>` HTML elements.
+
+For example, the following HTML provided by the user:
+
+```html
+<ff-suggest layout="block">
+    <section>
+        <div data-container="suggestType">
+            <span>Suggestions for suggestType</span>
+            <div>
+                <ff-suggest-item type="suggestType">
+                    <span>{{{name}}}</span>
+                </ff-suggest-item>
+            </div>
+        </div>
+    </section>
+</ff-suggest>
+```
+
+for 2 suggestions of ```suggestType``` will be rendered as:
+
+```html
+<ff-suggest layout="block">
+    <div style="position:relative;width:100%">
+        <div class="ffw-suggestContainerWrapper">
+            <div class="ffw-suggestContainer ffw-blockLayout">
+                <section>
+                    <div data-container="suggestType">
+                        <span>Suggestions for suggestType</span>
+                        <div>
+                            <ff-suggest-item type="suggestType">
```

Comment:
Mh, from comment in src code I would guess, that _"suggestType"_ is only a placeholder. I think we should list the possible types somewhere and create a real example without placeholder. Because it wasn't done before, feel free to decide, if you want to do it in this PR or if I should create a new ticket for that
```
/**
             * The "<b>type</b>" attribute specifies the type of the element. Valid type values are:<br>
             * <ul>
             *     <li><b>searchTerm</b><br></li>
             *     <li><b>brand</b><br></li>
             *     <li><b>productName</b><br></li>
             *     <li><b>category</b><br></li>
             *     <li><b>all</b></li>
             * </ul>
```

Reply:
As far as I know the set of suggest types is not closed. These are predefined values - user can add nev suggest type in FACT-Finder and map it to DB field. The types you found in source code comment are listed in ff-suggest-item documentation. If you want this section of documentation to be updated in details, please create a new ticket as this requires some more time.